sui-execution/latest/sui-move-natives/src/crypto/ecdsa_k1.rs

```diff
@@ -286,3 +293,74 @@ pub fn secp256k1_verify(
 
     Ok(NativeResult::ok(cost, smallvec![Value::bool(result)]))
 }
+
+pub fn secp256k1_sign(
+    _context: &mut NativeContext,
+    ty_args: Vec<Type>,
+    mut args: VecDeque<Value>,
+) -> PartialVMResult<NativeResult> {
+    debug_assert!(ty_args.is_empty());
+    debug_assert!(args.len() == 3);
+
+    // This function is only used for testing, so we don't need to charge gas
```

Comment:
is this just for now? I do not understand what it means for a public function to assume it is used only in testing